RecoTauTag/HLTProducers/src/CaloTowerFromL1TCreatorForTauHLT.cc

```diff
@@ -29,9 +29,10 @@ CaloTowerFromL1TCreatorForTauHLT::CaloTowerFromL1TCreatorForTauHLT(const Paramet
   produces<CaloTowerCollection>();
 }
 
-CaloTowerFromL1TCreatorForTauHLT::~CaloTowerFromL1TCreatorForTauHLT() {}
-
 void CaloTowerFromL1TCreatorForTauHLT::produce(StreamID sid, Event& evt, const EventSetup& stp) const {
+  if (mCone < 0.)
+    return;
```

Comment:
Same comment as above; the plugin should put an empty collection in the event before `return`ing.